src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -355,14 +372,14 @@ module TacoDependencyInstaller {
 
             return installerUtils.promptUser(resources.getString("YesExampleString"))
                 .then(function (answer: string): Q.Promise<any> {
-                    if (answer === resources.getString("YesString")) {
+                    if (answer.toLocaleLowerCase() === resources.getString("YesString")) {
```

Comment:
Telemetry is using this library for this kind of y/n: https://www.npmjs.com/package/readline-sync#utility_methods-keyinynstrict I think it'd be great if we could change all the code to use that same library to unify the whole source code.